Update hod.py

zaidikumail opened the pull request:
Added the Leauthaud+ 11 HOD class in hod.py

steven-murray commented:
Hi @zaidikumail -- sorry for not getting to this sooner! I appreciate the PR. It looks good, however, it might require some tidying up with `ruff`, and it would be great if you could add a test to cover it. Let me know if you need any help with those things!

zaidikumail commented:
@steven-murray I have added a test for the `Leauthaud11` HOD in `test_hod` which passes in a local install of this branch. Could you please take care of linting with `ruff` as I have other linters in place locally, and it might be much simpler for you to handle? As a side note, I noticed and fixed a syntax error in `setup.cfg` which popped up while pip installing this branch. Maybe this is already fixed in your `main` branch. Specifically, I added the missing comma in `camb>1.0.0<2.0` to make it `camb>1.0.0,<2.0`.
